addon/query-cache.js

```diff
@@ -5,7 +5,7 @@ import { assign } from '@ember/polyfills';
 
 import MegamorphicModel from './model';
 import M3QueryArray from './query-array';
-import { CUSTOM_MODEL_CLASS } from './feature-flags';
+import { CUSTOM_MODEL_CLASS } from 'ember-m3/-infra/features';
```

Comment:
p3/3 this import should actually resolve to a real path, as it does in master and on ember data.

Right now `-infra/features` seems to be purely synthetic.

Reply:
I removed the real path as it was unnecessary

Reply:
@runspired the real path should exist, as it does in ember data and ember.  Without it, the code ends up being much harder to follow when it has "phantom" imports.